build/build.sln

```diff
@@ -777,6 +773,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "AddressBook", "AddressBook"
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "AddressBook.UnitTests", "Tests\UnitTests\AddressBook\AddressBook.UnitTests.vcxproj", "{62E53898-65C2-4401-BF58-FBFB728E1B27}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "3rdParty", "3rdParty", "{DA66E27D-9C75-423E-BC9A-78D380FDB99D}"
```

Comment:
>}") [](start = 46, length = 3)

We really shouldn't have to add this to build.sln, it should go to libdispatch.sln.